src/components/LaboratoryChrome.js

```diff
@@ -1,103 +1,159 @@
+                </svg>
+              </a>
+              <span className="so-logo__separator"> </span>
+              <a href="#" className="so-logo__subSite">
+                Laboratory
+              </a>
+            </span>
+            <NetworkPicker />
+          </div>
         </div>
       </div>
-    </div>
-    <div className="so-back LaboratoryChrome__siteNavBack">
-      <div className="so-chunk">
-        <nav className="s-buttonList">
-          {tabItem('Introduction', SLUG.HOME)}
-          {tabItem('Create Account', SLUG.ACCOUNT_CREATOR)}
-          {tabItem('Explore Endpoints', SLUG.EXPLORER)}
-          {tabItem('Build Transaction', SLUG.TXBUILDER)}
-          {tabItem('Sign Transaction', SLUG.TXSIGNER)}
-          {tabItem('Submit Transaction', SLUG.TXSUBMITTER)}
-          {tabItem('View XDR', SLUG.XDRVIEWER)}
-        </nav>
+      <div className="so-back LaboratoryChrome__siteNavBack">
+        <div className="so-chunk">
+          <nav className="s-buttonList">
+            {tabItem("Introduction", SLUG.HOME)}
+            {tabItem("Create Account", SLUG.ACCOUNT_CREATOR)}
+            {tabItem("Explore Endpoints", SLUG.EXPLORER)}
+            {tabItem("Build Transaction", SLUG.TXBUILDER)}
+            {tabItem("Sign Transaction", SLUG.TXSIGNER)}
+            {tabItem("Submit Transaction", SLUG.TXSUBMITTER)}
+            {tabItem("View XDR", SLUG.XDRVIEWER)}
+          </nav>
+        </div>
       </div>
-    </div>
 
-    {getContent(props.routing.location)}
-    <RouterListener />
-  </div>;
+      {renamed(props.routing.location)}
+      <RouterListener />
+    </div>
+  );
 }
 
-function getContent(slug) {
+const Loading = ({ pastDelay, retry, error }) => {
+  if (error) {
+    return (
+      <div className="so-back">
+        <div className="so-chunk">
+          Error! <button onClick={retry}>Retry</button>
+          <pre>{error.stack}</pre>
+        </div>
+      </div>
+    );
+  } else if (pastDelay) {
+    return (
+      <div className="so-back">
+        <div className="so-chunk">loading…</div>
+      </div>
+    );
+  } else {
+    return null;
+  }
+};
```

Comment:
Oh, this is worth calling out as well. Since this PR changes features to load dynamically, we need a loading state. By default it waits 200ms before showing anything, so on fast networks this will be transparent to users.